world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
+skill_manager['hide'].type = 'skill'
+skill_manager['hide'].description = 'Allows a player to hide'
+skill_manager['hide'].level = 1
+skill_manager['hide'].hp_cost = 0
+skill_manager['hide'].mana_cost = 10
+skill_manager['hide'].move_cost = 0
+skill_manager['hide'].modifier = 0
+skill_manager['hide'].msg_to_caller = 'You slip into the shadows.'
+skill_manager['hide'].msg_to_room = '*caller* slips into the shadow.'
+skill_manager['hide'].msg_to_target = ''
+skill_manager['hide'].use_delay = 1
+skill_manager['hide'].affects = {}
+
+# kick skill
+skill_manager['kick'] = Ability()
+skill_manager['kick'].name = 'kick'
+skill_manager['kick'].type = 'skill'
+skill_manager['kick'].level = 1
+skill_manager['kick'].description = 'Allows a player to kick'
+skill_manager['kick'].hp_cost = 0
+skill_manager['kick'].mana_cost = 0
+skill_manager['kick'].move_cost = 10
+skill_manager['kick'].modifier = 0
+skill_manager['kick'].msg_to_caller = 'You kick *target*.'
+skill_manager['kick'].msg_to_room = '*caller* kicks *target*.'
+skill_manager['kick'].msg_to_target = '*caller* kicks you.'
+skill_manager['kick'].use_delay = 1
+skill_manager['kick'].affects = {}
+
+def show_skill_msg(mode=None,skill_name=None,caller=None,target=None):
+    """
```

Comment:
This can be a method on Ability. Given the code it shares with `show_spell_msg`, I think we can make this more generic to handle spells as well, and then remove `show_spell_msg`